engines/docker/generated_configschema.go

```diff
@@ -0,0 +1,47 @@
+// This source code file is AUTO-GENERATED by github.com/taskcluster/jsonschema2go
+
+package docker
+
+import "github.com/taskcluster/taskcluster-worker/runtime"
+
+type (
+	// Config applicable to docker engine
+	Config struct {
+
+		// Root Volume blah blah
+		RootVolume string `json:"rootVolume"`
+	}
+)
+
+func ConfigSchema() runtime.CompositeSchema {
+	schema, err := runtime.NewCompositeSchema(
+		"config",
+		`
+		{
+		  "$schema": "http://json-schema.org/draft-04/schema#",
+		  "additionalProperties": false,
+		  "description": "Config applicable to docker engine",
+		  "properties": {
+		    "rootVolume": {
+		      "description": "Root Volume blah blah",
+		      "title": "Root Volume",
+		      "type": "string"
+		    }
+		  },
+		  "required": [
+		    "rootVolume"
+		  ],
+		  "title": "Config",
+		  "type": "object"
```

Comment:
> We could rename taskcluster-base-go to just 'go'
I don't oppose that... But... it breaks the github fork/PR model..
If we really want this I propose calling it tc-go-libs...

------

But honestly, I don't see the need for us to have a collection of go libraries...
For small things I propose that we:
 1) put them in `taskcluster-client-go/tcutil/`
 2) put them in `taskcluster-worker/runtime/`
 3) Make then well documented and put them in `<yourname>/<packagename>` or `taskcluster/<packagename>` depending on how generic they are...

I think (1) and (2) will cover all the small stuff. (3) makes sense for things that are reusable outside the taskcluster project...